### PR TITLE
Production builds

### PR DESCRIPTION
**Pull request by alexfigtree**
Now with working video!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -201,11 +202,10 @@
     "stylelint-config-standard": "^18.2.0",
     "uglifyjs-webpack-plugin": "1.2.7",
     "url-loader": "^1.0.1",
-    "webpack": "^4.16.3",
-    "webpack-bundle-analyzer": "^2.13.1",
-    "webpack-cli": "^3.1.0",
-    "webpack-dev-server": "^3.1.5",
-    "webpack-merge": "^4.1.3",
+    "webpack": "^3.10.0",
```
Comment (jdpigeon) [inline package.json]: Had to downgrade, eh? Why is that?

File: webpack.config.base.js [hunk 2]
```diff
@@ -4,35 +4,10 @@
 
 import path from 'path';
 import webpack from 'webpack';
-import fs from 'fs';
 import { dependencies as externals } from './app/package.json';
-import { dependencies as possibleExternals } from './package.json';
-
-// Find all the dependencies without a `main` property and add them as webpack externals
-function filterDepWithoutEntryPoints(dep: string): boolean {
```
Comment (jdpigeon) [inline webpack.config.base.js]: Removing this function makes the entire app treated as webpack externals?

File: webpack.config.renderer.dev.dll.js [hunk 3]
```diff
@@ -20,20 +18,163 @@ export default merge.smart(baseConfig, {
...
-
   target: 'electron-renderer',
 
   externals: ['fsevents', 'crypto-browserify'],
 
   /**
    * Use `module` from `webpack.config.renderer.dev.js`
    */
-  module: require('./webpack.config.renderer.dev').module,
+  module: {
```
Comment (jdpigeon) [inline webpack.config.renderer.dev.dll.js]: Does this replace the bundling that would have occurred on code previously not treated as a webpack external?

File: webpack.config.renderer.dev.js [hunk 4]
```diff
@@ -59,20 +55,6 @@ export default merge.smart(baseConfig, {
 
   module: {
     rules: [
-      {
```
Comment (jdpigeon) [inline webpack.config.renderer.dev.js]: Why was this removed?

File: package.json (hunk 1 above)
Comment (alexfigtree) [reply package.json]: Yes! I had to downgrade because because I needed to install 'extract-text-webpack-plugin' to get this to work, and it requires an older version of webpack. Things broke when I tried using the most recent version. :-/

File: webpack.config.base.js (hunk 2 above)
Comment (alexfigtree) [reply webpack.config.base.js]: Not sure, I replaced the file with what was working in 'erb-video-example', but I just re-tested this without these changes and there was no problem. Check out a latest commit reverting these changes.
